typing/jkind.mli

```diff
@@ -180,8 +187,14 @@ end
 (** Context for jkind operations. *)
 type jkind_context =
   { jkind_of_type : Types.type_expr -> Types.jkind_l option;
-    is_abstract : Path.t -> bool
+    is_abstract : Path.t -> bool;
         (* Check if a type path refers to an abstract type *)
+    lookup_type : Path.t -> Types.type_declaration option;
+    normalize_path : Path.t -> Path.t;
```

Comment:
What does this do? And why do we need it?

Reply:
Looking at the code, it seems that `normalize_path` is really part of type expansion. It strikes me as unusual to use it where you are, unless you're also doing type expansion. For example, given

```
module M1 = struct
  type t
end

module M2 = M1

type t2 = M1.t
```

then (I think) normalizing paths will tell you that `M2.t` is the same as `M1.t`, but will fail to tell you that `t2` is the same as `M.t`. Perhaps normalizing paths will be a nice little optimization (in that more types can be recognized as equal), but my guess is that it's not needed. `Path.same` seems to be the best way to equate paths.